nova/conf/scheduler.py

```diff
@@ -423,6 +423,27 @@
 * An integer or float value, where the value corresponds to the multipler
   ratio for this weigher.
 """),
+    cfg.FloatOpt("prefer_same_shard_resize_weight_multiplier",
+        default=1.0,
+        help="""
+Prefer scheduling on same-shard on resize weight multiplier.
+
+This option determines how strongly the previous shard should be
+considered for scheduling a resizing instance. A positive value will result in
```

Comment:
I would say "preferred" instead of "considered".

Reply:
fixed